kubernetes.yaml

```diff
@@ -80,6 +85,15 @@ data:
 
     repository   = scrapyd_k8s.repository.Remote
     launcher     = scrapyd_k8s.launcher.K8s
+    
+    num_lines_to_check = 2
+    
+    storage_provider = s3
+    
+    aws_region            = <aws_region>
+    aws_access_key_id     = <aws_access_key_id>
+    aws_secret_access_key = <aws_secret_access_key>
+    aws_bucket_name       = <aws_bucket_name>
```

Comment:
Good to see something in the configuration file. I think this warrants its own section, maybe `logging` but that could also refer to the daemon's logging settings, so perhaps something like `[logwatcher]` or `[logging.jobs]` or something.

Reply:
Or if we want to make this project-specific as well, then we'd perhaps use `[default.logs]` and `[<project>.logs]` or so.